js/core/webidl-contiguous.js

```diff
@@ -641,8 +641,10 @@ define(
                     case "enum":
                         name = defn.name;
                         defn.values.filter(function (v) { return v.type === undefined;})
-                                   .forEach(function(v) {
-                            v.dfn = findDfn(name, v, definitionMap, msg);
+                        .map(function(v) {
```

Comment:
Won't "map" return a new array? Where is that array stored?
I would expect something like defn.values = defn.values.filter().map(function(v) {...});